specification/components/schemas/SearchCriteriaParameters.yaml

```diff
@@ -0,0 +1,46 @@
+title: Wraps the criteria used to search for services for a related referral
```

Comment:
I think this title could read a bit better, maybe simply "The criteria used to search for services".